docs/www/kubernetes-external-connect.md

```diff
@@ -0,0 +1,150 @@
+---
+title: How do I connect to a Kubernetes cluster?
+order: 0
+---
+
+# How do I connect to a Kubernetes cluster?
+
+The [Kubernetes Quick Start guide](quick-start-kubernetes.md) describes how to quickly get up and running with a Kubernetes cluster.
+Those instructions only provide access to the cluster from within the Kuberenetes network.
+
+Here we'll show you an example of how to set up Kubernetes so you can work with Redpanda from outside of the Kubernetes network.
+
+Let's get started...
+
+## Create a Kubernetes cluster
+
+
+
+1. Create three node cluster with enough CPUs
+
+<tabs>
+
+  <tab id="Kind">
+
+  [Kind](https://kind.sigs.k8s.io) is a tool that lets you create local Kubernetes clusters using Docker.
+    After you install Kind, set up a 3-node cluster with:
+
+  ```
+  cat > 3-node-cluster.yaml <<EOF
+  kind: Cluster
+  apiVersion: kind.x-k8s.io/v1alpha4
+  nodes:
+  - role: control-plane
+  - role: worker
+  - role: worker
+  EOF
+  kind create cluster --config 3-node-cluster.yaml
+  ```
+
+  </tab>
+
+  <tab id="AWS EKS">
+
+  Use the [EKS Getting Started](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) guide to set up EKS.
+  When you finish, you'll have `eksctl` installed so that you can create and delete clusters in EKS.
+  Then, create an EKS cluster with:
+
+  ```
+  eksctl create cluster \
+  --name redpanda \
+  --nodegroup-name standard-workers \
+  --node-type m5.xlarge \
+  --nodes 3 \
+  --nodes-min 1 \
+  --nodes-max 4 \
+  --node-ami auto
+  ```
+
+  It will take about 10-15 minutes for the process to finish.
+
+  </tab>
+
+  <tab id="Google GKE">
+
+  First complete the "Before You Begin" steps describe in [Google Kubernetes Engine Quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+  Then, create a cluster with:
+
+  ```
+  gcloud container clusters create redpanda --machine-type e2-standard-4
+  ```
+
+  **_Note_** - You may need to add a `--region`, `--zone`, or `--project` to this command.
+
+  </tab>
+</tabs>
+
+### Make it possible to connect to cluster from kubectl
+
+```
+gcloud container clusters get-credentials redpanda
+```
+
+### Install cert-manager
+
+```bash
+kubectl create namespace cert-manager && \
+helm repo add jetstack https://charts.jetstack.io && \
+helm repo update && \
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.2.0 \
+  --set installCRDs=true
```

Comment:
The flag in helm `--create-namespace` make the `kubectl create namespace` obsolete
```suggestion
helm repo add jetstack https://charts.jetstack.io && \
helm repo update && \
helm install \
  cert-manager jetstack/cert-manager \
  --namespace cert-manager \
  --create-namespace \
  --version v1.2.0 \
  --set installCRDs=true
```